modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1575,7 +1575,7 @@ Please use a path with a length of 1 [r*1..1] instead or a Match with a limit.
 - The query used a deprecated function. (`%s`)
 - The query used a deprecated procedure. (`%s`)
 - The query used a deprecated runtime option. (`%s`)
-- The `TextIndexProvider.DESCRIPTOR.name()` provider for text indexes is deprecated and will be removed in a future version.
+- `text-1.0`, `text-2.0` providers for text indexes are deprecated and will be removed in a future version. Please use `text-3.0` instead.
 Please use `TrigramIndexProvider.DESCRIPTOR.name()` instead.
```

Comment:
Should we keep the line 1579?

Reply:
oj smt happened with that in the inline editor from the github and line was resurrected somehow. 
It should be removed, cleaning up.